src/demo/pages/BlockchainTransaction/SignTransactionSendPayment.tsx

```diff
@@ -48,20 +48,15 @@ const SignTransactionSendPayment = (props: SignTransactionSendPaymentProps) => {
       const accountName = await user?.getAccountName();
 
       if (accountName) {
-        let accountBalance = await eosioTokenContract.getBalance(accountName);
+        const accountBalance = await eosioTokenContract.getBalance(accountName);
 
         props.setBalance(accountBalance);
-        setAmount(Math.floor(accountBalance / 2));
```

Comment:
this doesnt look right. The amount to send is not the same as the current balanec. This formular here set the amount to send as half the current balance. It should be less than the current balanec. I think the deleted logic needs to stay.